### PR TITLE
fix [2013] change num of decimals displayed

### PR DESCRIPTION
**Pull request by gwan93**
Ref: https://trello.com/c/Uqc7cE4o/2013-macroinvertrebrate-summary-table-needs-to-use-the-same-number-of-decimal-places

Note: the summary looks a little different because another PR with the format changes is waiting to be reviewed

image

---Every PR---

- [x] Changes have been tested locally
- [x] Lint has run and any errors have been resolved
- [x] Prettier has run on any changed files
- [x] Unit tests have run and passed

---Transitional Changes---

- [x] Any hard-coded text in the file(s) worked has been refactored into key-value tokens
- [x] Any language.js tokens no longer used are removed
- [x] Any necessary updates to the documentation have been made
- [x] Unit tests have been added or updated, where possible, to prevent future regressions
- [x] styled-components code in changed files has been updated to CSS modules in the styles folder
- [x] Touched JS files are migrated to TypeScript where in scope; PropTypes are replaced with TypeScript interfaces in converted files and not added for shapes already typed in `mermaidDataTypes.ts`.

---Post-Merge---

- [ ] Code has merged and build success is confirmed in 'Actions'
- [ ] The corresponding Trello ticket has moved to the appropriate list (likely User-QA)

### CODE REVIEW

**Review by alanjleonard — APPROVED** [review]

File: src/library/macroinvertebrates/useBeltInvertDensityMetrics.ts [hunk 1]
```diff
@@ -52,8 +52,8 @@ interface BeltInvertObservationMetrics {
   observationDensities: Map<string, number>
 }
 
-export const formatDensityToTwoDecimals = (value: number) =>
-  (Math.round(Number(value) * 100) / 100).toFixed(2)
+export const formatDensityToOneDecimal = (value: number) =>
+  (Math.round(Number(value) * 100) / 100).toFixed(1)
```
Comment (JBurkinshaw) [inline src/library/macroinvertebrates/useBeltInvertDensityMetrics.ts]: When this rounded to two decimals the `Math.round(value * 100) / 100` matched the `.toFixed(2)` and was fine. Now that we output one decimal, it double-rounds: it rounds to two decimals first, then `.toFixed(1)` rounds again. That shifts some values, e.g. `1.249` → `1.3` (should be `1.2`), `1.45` → `1.4` (should be `1.5`), `2.349` → `2.4` (should be `2.3`).

We already have `roundToOneDecimal` in `library/numbers`, which does the correct single-step round, and both files in this PR already import it (used a few lines away).

I suggest deleting `formatDensityToOneDecimal` and calling `roundToOneDecimal` at the four call sites instead. That fixes the rounding and drops a near-duplicate helper. (Values here are typed `number`, so the missing `Number()` wrapper in `roundToOneDecimal` isn't an issue.)

Note that `roundToOneDecimal` has no unit test, so this consolidation doesn't add coverage. Worth a small `roundToOneDecimal.test.js` pinning the boundary cases (`1.249 → "1.2"`, `1.45 → "1.5"`) so the display precision is protected going forward. Something along these lines:
```js
import { roundToOneDecimal } from './roundToOneDecimal'

describe('roundToOneDecimal', () => {
  it.each([
    [1.249, '1.2'],
    [1.45, '1.5'],
    [2.349, '2.3'],
    [0, '0.0'],
  ])('rounds %f to %s', (input, expected) => {
    expect(roundToOneDecimal(input)).toBe(expected)
  })
})
```

**Review by JBurkinshaw — COMMENTED** [review]
Correct intent, minimal and well-scoped. The one thing I'd change before merge is dropping the redundant helper in favour of `roundToOneDecimal` (see comment in code) - it fixes a rounding discrepancy and removes duplication, with essentially no added risk since the helper is already imported in both files

**Review by JBurkinshaw — APPROVED** [review]
